src/Model/HasLegacyAttributesTrait.php

```diff
@@ -64,6 +76,10 @@ public function getAttribute(string $key)
     {
         $inAttributes = array_key_exists($key, $this->attributes);
 
+        if ($caster = CastResolver::resolve($this->casts[$key] ?? null)) {
```

Comment:
The same solution described before can be used here.

Reply:
Done: https://github.com/leroy-merlin-br/mongolid/pull/220/commits/0fa4607f1c7a1c45c9c09bd8411843350ac6dbe7